### ISSUE TITLE
aachener-zeitung.de

### ISSUE THREAD

**Issue by wanhose** [wanhose/cookie-dialog-monster#713]
#### Browser
Firefox (129.0)
#### Reason
Popup showed up
#### URL
https://www.aachener-zeitung.de/
#### Version
7.0.4

**Comment by dreirund** [comment]
Despite that the addon's icon shows a number "1" badge:

---

![image](https://github.com/user-attachments/assets/1aa0d519-ad25-45ba-a956-91738c91e431)

**Comment by wanhose** [comment]
Hi @dreirund,

This should be fixed now, if not please wait until `7.1.0` is published.

Thanks again 😄

**Comment by dreirund** [comment]
I did install version 7.1.0 of this addon, updated the database, and opened the URL again (in the background).

I got the popup again. The addon's icon does show a "1" badge.

When I switch to it's tab and reload, the popup is removed and the addon's icon shows a "2" badge.

I can reproduce it:

* Opening in the background &rarr; popup is there and stays.
* Opening/ reloading in the foreground &rarr; popup is removed.

---

![image](https://github.com/user-attachments/assets/d866e085-5246-4be6-888c-19062ec2aeb0)
